app/Http/Controllers/DocumentsController.php

```diff
@@ -46,38 +45,38 @@ public function upload(Request $request, $id)
         Session::flash('flash_message', 'File successfully uploaded');
     }
 
-    /**
-     * @param Request $request
-     *
-     * @return mixed
-     */
-    public function import(Request $request)
-    {
-        $rules = [
-            'file'        => 'required',
-            'num_records' => 'required',
-        ];
-        $validator = Validator::make($request->all(), $rules);
-        // process the form
-        if (!$validator->fails()) {
-            return Redirect(route('clients.create'))->withErrors($validator);
-        } else {
-            try {
-                Excel::load('public\imports\contacts.xlsx', function ($reader) {
-                    foreach ($reader->toArray() as $row) {
-                        if ($row['name'] && $row['company'] && $row['email'] && '' == $row['user']) {
-                            Session::flash('flash_message_warning', 'Required fields are empty');
-                        }
-                    }
-                });
-                Session::flash('flash_message', 'Users uploaded successfully.');
-                // return redirect(route('clients.index'));
-            } catch (\Exception $e) {
-                Session::flash('flash_message_warning', $e->getMessage());
-                //return redirect(route('clients.index'));
-            }
-        }
-    }
+    // /**
```

Comment:
Why is this outcommented?

Reply:
I didn't see where this import class was actually used in the app. Perhaps I missed something? Also, PHPExcel stopped working at some point during the series of version upgrades. Since it didn't look like it was actually used, I chose to comment it out rather than try to troubleshooting.

My intention is to move to box/spout for Excel and CSV import/export. Mostly because I'm very familiar with Spout. I use it every day in my work.